content/en/blog/v2-release/index.md

```diff
@@ -0,0 +1,35 @@
+{
+    "title": "Datenanfragen v2: Refreshed interface",
+    "type": "blog",
+    "date": "2022-12-06T12:00:00+02:00",
+    "description": "Today we release the next version of our website with many improvements: A new face for request generator, more comprehensive request history, a fancy new frontpage and a generator for complaints and admonitions.",
+    "featured_image": "",
+    "tags": [ "complaint generator", "new user experience", "update", "open source" ],
+    "authors": [ "zner0L" ]
+}
+
+In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to comapnies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).
```

Comment:
```suggestion
In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to companies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).
```

Reply:
comapnies -> companies

Not sure why that isn't being highlighted.